Cargo.toml

```diff
@@ -660,6 +660,7 @@ backtrace = { git = "https://github.com/rust-lang/backtrace-rs.git", rev = "7226
 color-eyre = { git = "https://github.com/eyre-rs/eyre.git", rev = "e5d92c3" }
 deltalake = { git = "https://github.com/delta-io/delta-rs", rev = "9954bff" }
 display-more = { git = "https://github.com/databendlabs/display-more", tag = "v0.2.0" }
+jsonb = { git = "https://github.com/b41sh/jsonb", rev = "8d82534b70c463ac4f42fb2799d41bd6559c174e" }
```

Comment:
Why we use the `https://github.com/b41sh/jsonb` version?

Reply:
This is just for test, has been updated to the new jsonb version